nncf/quantization/algorithms/smooth_quant/backend.py

```diff
@@ -82,12 +82,12 @@ def get_channel_agnostic_reduction_shape(channel_axis: int, shape: Tuple[int]) -
     @staticmethod
     @abstractmethod
     def get_abs_max_channel_collector(
-        num_samples: int, stats_reduction_shape: Tuple[int], inplace: bool, branch_key: str
+        num_samples: int, stats_reduction_axes: Tuple[int], inplace: bool, branch_key: str
     ) -> TensorCollector:
         """
         Returns TensorCollector with MaxAggregator and AbsMaxReducer.
 
-        :param stats_reduction_shape: Calculated reduction shape.
+        :param stats_reduction_axes: Calculated reduction shape.
```

Comment:
It might be good if we change `reduction shape -> reduction axes` everywhere in this PR. Even in the docstrings.

Reply:
Done